packages/file/lib/src/backends/memory/memory_random_access_file.dart

```diff
@@ -0,0 +1,377 @@
+// Copyright (c) 2020, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:convert';
+import 'dart:math' show min;
```

Comment:
nit: `as math show min;`

Reply:
I was following what `memory_file.dart` does.  Do you want me to change it there too?

Reply:
Sure. I'm probably biased towards the Flutter conventions, which this repo might not be fully following but seems to mostly.